book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
A method for adding rentals should be present, and Inside this method the book itself should be sent as a parameter to create the new rental.